pkg/lightning/log/filter_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+package log_test
+
+import (
+	"regexp"
+	"strings"
+
+	. "github.com/pingcap/check"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/pingcap/br/pkg/lightning/log"
+)
+
+var _ = Suite(&testFilterSuite{})
+
+type testFilterSuite struct{}
+
+func (s *testFilterSuite) TestFilter(c *C) {
```

Comment:
how about test a `NameError` or something, which will print stacks that contain the specified package name

Reply:
Could you show an example?

Reply:
maybe

```
	// stack messages won't trigger filter
	logger, buffer = log.MakeTestLogger(zap.WrapCore(func(c zapcore.Core) zapcore.Core {
		return log.NewFilterCore(c, "github.com/pingcap/check").With([]zap.Field{zap.String("a", "b")})
	}), zap.AddCaller())
	logger.Warn("the message", zap.NamedError("stack", errors.New("test")))
	c.Assert(strings.Contains(buffer.Stripped(), "github.com/pingcap/check"), IsTrue)
```